assets/js/gamepage.js

```diff
@@ -31,7 +31,7 @@ var score = 0;
     } else if(user_val > comp_val ) {
       jokeBoxEl.html('');
       jokeModal.modal( 'open' );
-      score +=user_val;
+      localStorage.setItem("score", JSON.stringify(score +=user_val));
```

Comment:
There is no need to modify the actual game code. This line should be removed and changed back to the previous one